_scripts/check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+if [[ $(which docker) && $(docker --version) ]]; then
+  docker=y
+else
+  docker=n
+fi
+
+os="$(uname -a | cut -f 1 -d ' ')"
+if [ "$os" = "Darwin" ]; then
+    if [ "$docker" = "n" ]; then
+      echo "install docker to continue installation"
+      echo "https://docs.docker.com/docker-for-mac/install/"
+      exit 1
+    fi
+
+    if [[ $(which brew) && $(brew --version) ]]; then
+      brew install gmp graphicsmagick
```

Comment:
and we can also remove `brew install gmp graphicsmagick` from the README now that we have it here